lib/utils.pm

```diff
@@ -1291,4 +1293,20 @@ sub reconnect_s390 {
     }
 }
 
+sub kernel_packages {
```

Comment:
there is a high chance that this function would be used exclusively by kernel tests. what if you will create kernel-utils and move it there?  current lib/utils is too big. I already did attempts to solve this by introducing version_utils, but there is still a lot of place for improvement. So let's at least not increase this big file with something which is used only in one file or something which used in some certain area ( e.g. kernel, nfv  etc. )